pkg/api/types.go

```diff
@@ -504,20 +505,31 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	isWindows, _ := regexp.MatchString("^[A-Za-z]:\\.*", value)
```

Comment:
Apparently this method `Set` is called in a loop. It would not be a bad idea to compile the regexp only once at initialization time. Also, in general, ignoring errors like here raises a red flag.

Suggestion:

```go
var someName = regexp.MustCompile(`...`)
```